package.json

```diff
@@ -277,7 +284,7 @@
     "test:e2e": "echo Not implemented",
     "test": "yarn run test:unit && yarn run test:e2e",
     "build-mod-descriptor": "stripes mod descriptor --full --strict | jq '.[]' > module-descriptor.json ",
-    "formatjs-compile": "stripes translate compile"
+    "formatjs-compile": "formatjs compile-folder --ast --format simple ./translations/ui-bulk-edit ./translations/ui-bulk-edit/compiled"
```

Comment:
Could you please clarify why we need this change?

Reply:
Agreed; this seems like a step backwards. If there's a problem with `stripes translate`, please LMK so we can fix it! 

Related, we forgot to remove the dep on `@formatjs/cli` in PR #696 when we migrated to `stripes translate`. There's no harm, it's just cruft, but the reason `stripes translate` exists is to lift up those deps into stripes-cli (so you don't have to worry about them) and to provide simple and consistent commands in `package.json` :)